tools/cnvkit_import_theta2.cwl

```diff
@@ -10,69 +10,76 @@ requirements:
     ramMin: 8000
     coresMin: 4
 
-baseCommand: [cnvkit.py, import-theta]  
+baseCommand: []  
 arguments:
   - position: 1
     shellQuote: false
     valueFrom: >- 
-      $(inputs.tumor_cns.path)
-      $(inputs.theta2_n2_results.path)
+      ${
+        if (inputs.theta2_n2_results == null){
+          return "echo Theta 2 input is null, skipping purity adjustment >&2; exit 0;"
+        }
+      }
+
+      cnvkit.py import-theta
+      ${if (inputs.tumor_cns != null) {return inputs.tumor_cns.path;}}
```

Comment:
`tumor_cns` is not optional and this program (judging by its docs) will fail if it is not provided. Wondering why it receives the same treatment as the `theta2_n2_results`.

Reply:
Hmm, yeah, good point.  I think it was a panic statement in that others would fail if i didn't wrap it, so i blindly wrapped the others.  I'll remove that, and test it out real quick just to make sure that logic holds.

Reply:
Made the change, tried it out real quick, nothing broke